lib/active_record/virtual_attributes/virtual_delegates.rb

```diff
@@ -125,7 +126,7 @@ def #{method_name}(#{definition})
           module_eval(method_def, file, line)
         end
 
-        def virtual_delegate_name_prefix(prefix, to)
+        def virtual_delegate_name_prefix(prefix, to)  # rubocop:disable Naming/MethodParameterName
```

Comment:
I'm surprised this didn't trigger another rubocop with the 2 spaces before the comment.

Reply:
lol. it did trigger that